src/routes/api/email/+server.ts

```diff
@@ -1,29 +1,34 @@
 import mailchimp from '@mailchimp/mailchimp_transactional';
 import { env } from '$env/dynamic/private';
-import dynamics from '$lib/dynamics';
-import type { MessageRecipient } from '@mailchimp/mailchimp_transactional';
 
 const apiKey = env.MAILCHIMP_API_KEY;
 const client = mailchimp(apiKey ?? '');
 
 export async function POST({ request }) {
 	try {
-		const { email, results } = await request.json();
+		const { email, results_string } = await request.json();
 		// todo: include email validation
-		const features = Object.keys(results);
 
 		const message = {
-			to: [{ email, type: 'to' } as MessageRecipient],
-			from_email: 'info@allwecansave.earth',
-			subject: '8 Dynamics - Your Results',
-			html: `<h1>Your Results: </h1>
-				<div>
-				${dynamics.map((d, i) => `<span>${d.short}: ${results[features[i]]}</span>`)}
-				</div>
-			`
+			template_name: 'eight-dynamics',
+			template_content: [
+				{
+					name: 'results_link',
+					content: `Thank you for spending some time with the 8 Dynamics of Climate Engagement! Your results are <a href=${process.env.BASE_URL}/results/${results_string} style="color: #303326; text-decoration: underline">here</a>.`
```

Comment:
Yay thank you! I think you may just need quotes around the `href` (`<a href="____" ... `) otherwise looks great!